src/energyplus/ForwardTranslator/ForwardTranslateCoilSystemCoolingWater.cpp

```diff
@@ -74,36 +74,32 @@ namespace energyplus {
           idf->setString(Coil_Cooling_WaterFields::AirInletNodeName, airInletNodeName);
           idf->setString(Coil_Cooling_WaterFields::AirOutletNodeName, airOutletNodeName);
           // Add IddObjectType::Coil_Cooling_Water_DetailedGeometry if implemented
+        } else if (idf->iddObject().type() == IddObjectType::CoilSystem_Cooling_Water_HeatExchangerAssisted) {
+          // The logic is built in the translateCoilSystemCoolingWaterHeatExchangerAssisted method
+          // Essentially it's the same as getting the HX (CoilSystem_Cooling_Water_HeatExchangerAssistedFields::HeatExchangerName)
+          // and setting HeatExchanger_AirToAir_SensibleAndLatentFields::SupplyAirInletNodeName & ExhaustAirOutletNodeName
```

Comment:
We gotta handle the `CoilSystem_Cooling_Water_HeatExchangerAssisted` case. I'm not doing it here because it's too complicated/awkward because we work with IdfObject, not WorkspaceObject, so theree isn't a getTarget available to set the HX Nodes